examples/osgi-test-example-bnd/itest/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
+encoding//src/itest/ExampleIntegrationTest.java=UTF-8
+encoding/bnd.bnd=UTF-8
```

Comment:
`encoding/<project>=UTF-8` covers the whole project not just individual files.

Reply:
thx. done